python_modules/dagit/dagit/dagit_tests/test_multiprocessing.py

```diff
@@ -73,17 +83,24 @@ def test_failing():
     pipeline = define_failing_pipeline()
     config = {
         'solids': {
-            'load_num_csv': {
-                'config': {
-                    'path': script_relative_path('num.csv'),
+            'sum_solid': {
+                'inputs': {
+                    'num': {
+                        'csv': {
+                            'path': script_relative_path('num.csv'),
+                        },
+                    },
                 }
             },
         },
     }
-    typed_environment = evaluate_config_value(pipeline.environment_type, config)
+    typed_environment = create_typed_environment(pipeline, config)
```

Comment:
seeing this boilerplate again and again makes me think we want to write a test fixture to smooth this kind of thing.